FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -5218,6 +5218,9 @@ void OpDispatchBuilder::INTOp(OpcodeArgs) {
     constexpr uint8_t SYSCALL_LITERAL = 0x2E;
 #endif
     if (Literal == SYSCALL_LITERAL) {
+      if (CTX->Config.Is64BitMode()) [[unlikely]] {
+        LogMan::Msg::AFmt("[Unsupported] Trying to execute 32-bit syscall from a 64-bit process.");
```

Comment:
What will happen in this case? Is there a chance this will behave correctly at all?

Reply:
It will not behave correctly. It will call an incorrect syscall since we only install syscall handlers for the architecture that was originally started.
This will force a SIGABRT with the log message.

Reply:
Isn't `AFmt` a no-op on Release/RelWithDebInfo builds?

Reply:
`LOGMAN_MSG_A_FMT` is a no-op. `LogMan::Msg::AFmt` is not.

Reply:
er wait, made a fool. That was legacy behaviour. I need to change this

Reply:
Swapped to `ERROR_AND_DIE_FMT`

Reply:
I wasn't entirely sure either, but glad it's fixed now :)